.github/workflows/check_tests.yml

```diff
@@ -81,8 +81,91 @@ jobs:
           SCCACHE_DIR: /home/runner/.cache/sccache
         run: |
           source ${HOME}/.cargo/env
-          RUSTC_BOOTSTRAP=1 cargo test --release --features=runtime-benchmarks,try-runtime -- --test-threads=2
-      - name: run live benchmarks test
+          RUSTC_BOOTSTRAP=1 cargo test --release --features=runtime-benchmarks,try-runtime
```

Comment:
this no longer needs the `runtime_benchmarks` feature, right?
benchmark test will not use this compilation result anyway

Reply:
`runtime-benchmarks` feature is needed for the benchmarks unit tests and `release` is needed primarily for the MantaPay tests, in debug they run much slower.